fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/RecordMetaData.java

```diff
@@ -171,6 +171,16 @@ public RecordType getRecordTypeForDescriptor(@Nonnull Descriptors.Descriptor des
         return recordType;
     }
 
+    @Nonnull
+    public RecordType getRecordTypeFromRecordTypeKey(@Nonnull Object recordTypeKey) {
```

Comment:
This and `getIndexFromSubspaceKey` should probably document that they throw a `MetaDataException` if the object specified is not found in the meta-data.

Reply:
Added.